frontend/src/routes/Credentials/CreateCredentialsCatalog.tsx

```diff
@@ -70,13 +73,15 @@ export function CreateCredentialsCatalog() {
         />
       }
     >
-      <ItemView
-        items={cards}
-        itemKeyFn={keyFn}
-        itemToCardFn={(card) => card}
-        onBack={back(NavigationPath.credentials)}
-        onCancel={cancel(NavigationPath.credentials)}
-      />
+      <DataViewStringContext.Provider value={dataViewStrings || defaultStrings}>
```

Comment:
There's no need to pass in `defaultStrings` here or in any of the spots where you've added `<DataViewStringContext.Provider>`. `dataViewStrings` is never undefined.